src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const price = 40;
+  const basicDiscount = 50;
+  const additionalDiscount = 20;
+
+  if (days < 3) {
+    return days * price;
+  }
+
+  if (days >= 3 && days < 7) {
+    return (days * price) - additionalDiscount;
+  }
+
+  if (days >= 7) {
+    return (days * price) - basicDiscount;
+  }
```

Comment:
```suggestion
  if (days < 7) {
    return (days * price) - additionalDiscount;
  }

    return (days * price) - basicDiscount;

```

no need for extra checks, just be consistent